src/inc/msquicp.h

```diff
@@ -29,7 +29,31 @@ extern "C" {
 //
 // The different private parameters for QUIC_PARAM_LEVEL_GLOBAL.
 //
+
+typedef

```

Comment:
I think putting HOOK in the name will make it much more obvious what these are for. QUIC_TEST_DATAPATH_RECEIVE_HOOK_CALLBACK etc

Reply:
Sure. I can do that.